src/uu/install/src/install.rs

```diff
@@ -521,6 +525,21 @@ fn copy(from: &PathBuf, to: &PathBuf, b: &Behavior) -> Result<(), ()> {
         return Err(());
     }
 
+    if b.strip && cfg!(not(windows)) {
+        match Command::new(&b.strip_program).arg(to).output() {
+            Ok(o) => {
+                if !o.status.success() {
+                    crash!(
+                        1,
+                        "strip program failed: {}",
+                        String::from_utf8(o.stderr).unwrap_or_default()
```

Comment:
can you please trigger this error ?
for example, pass "ls" as strip program
(sorry I didn't notice earlier)

Reply:
No issues. Added a test case to check this.